src/config/configuration.ts

```diff
@@ -0,0 +1,10 @@
+import { config } from 'dotenv';
+import Iconfig from './iconfig';
+config();
+
+const configuration: Iconfig = Object.freeze( {
+    port: process.env.PORT,
```

Comment:
did you check whether this env is available here? As the .env file is not located at its right position

Reply:
Done